package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.55.0",
   "description": "Official monday.com UI resources for application development in React.js",
   "main": "./dist/main.js",
+  "types": "./dist/types.d.ts",
```

Comment:
@hadasfa if you're planning to merge this code before you guys finish migrating all the components (highly recommended!), don't forget to remove this line so it won't cause problems with existing MF using vibe

Reply:
why this can cause problems? can we talk about this?